src/settings/index.ts

```diff
@@ -98,27 +100,49 @@ export function useSettings(): [Settings, SettingsSetters] {
                 if (document.exists) {
                     const data = document.data()
 
-                    // Handle settings which were initialized before `new settings` was introduced.
-                    if (data.owners === undefined) {
-                        persistToFirebase(getDocumentId(), 'owners', [])
-                        data.owners = []
+                    const defaultSettings = {
+                        description: '',
+                        logoSize: '32px',
+                        theme: 'default',
+                        owners: [] as string[],
+                        ...data,
                     }
 
-                    if (data.theme === undefined) {
-                        persistToFirebase(getDocumentId(), 'theme', 'default')
+                    if (
+                        data.owners === undefined ||
+                        data.owners.length === 0 ||
+                        data.owners.includes(user?.uid)
+                    ) {
```

Comment:
Er det noen grunn til å nøste alle disse `if`'ene under en lengre `if`? Ser litt forvirrende ut. `data.owners === undefined` blir jo sjekket to ganger, f.eks.